components/design-system/scss/helpers/_helpers-print.scss

```diff
@@ -38,24 +26,30 @@
-      display: block;
-      text-align: center;
-      top: 0;
-      width: 100%;
-    }
-
     a,
     a:visited {
       text-decoration: underline;
     }
 
+    // Don't show links in headings
+    // stylelint-disable no-descending-specificity
+    h1 a,
+    h2 a,
+    h3 a,
+    h4 a,
+    h5 a,
+    h6 a {
+      @include nds-no-appended-print-link;
+    }
+
     a[href]:after {
       content: ' (' attr(href) ')';
     }
 
+    a.show-link[href]:after {
```

Comment:
Can you combine this with the rule above? ...but also, do we need it?

Reply:
Or maybe it should be called `append-print-link` or something?

Reply:
I personally can't imagein this being used - will remove